src/DocumentationBrowserViewExtension/DocumentationBrowserViewModel.cs

```diff
@@ -276,7 +277,30 @@ private string LoadContentFromResources(string name)
                 assembly = GetType().Assembly;
             }
 
-            var availableResources = assembly.GetManifestResourceNames();
+            // Resolve satellite assembly matching UI culture
+            var culture = CultureInfo.CurrentUICulture;
+            var resourceAssembly = assembly.GetSatelliteAssembly(culture);
+            if (resourceAssembly == null)
+            {
+                if (culture.IsNeutralCulture)
+                {
+                    var specificCulture = CultureInfo.CreateSpecificCulture(culture.Name);
+                    resourceAssembly = assembly.GetSatelliteAssembly(specificCulture);
+                }
+                else
+                {
+                    while (resourceAssembly == null && culture.Parent != CultureInfo.InvariantCulture)
```

Comment:
hmm, why is this a while loop? Will assembly or culture every be changed by the while loop?

Reply:
You found a bug. That's why I said it needed testing 😂 

In theory we shouldn't need a while loop. There are only two levels of cultures in .NET, beyond invariant which we don't really care about here. Maybe I can just remove the loop to avoid complexity.

Reply:
@mjkkirschner More importantly, what do you think of this approach? My feeling is that it's the natural extension to what we had, and not too complex on our side. I believe the localization team should find this approach easy to use.

Reply:
I propose we RC2.7.0 now and test this in master ASAP 👍

Reply:
Thanks @QilongTang ! I'll start working on the tests now